README.md

```diff
@@ -113,9 +113,9 @@ flowchart TB
     <summary>CLI options</summary>
 ec2.2 takes the following flags:
 
-- **region**: _Required._ The region you want to search in.
+- **profile**: _Required._ The profile to use when connecting to AWS. 
 
-- **profile**: _Optional._ The profile to use when connecting to AWS. If not
+- **region**: _Optional._ The region you want to search in. If not
```

Comment:
It might be good to update the command in line 87 as well, to make it explicit that the region is now optional and to encourage people to check all regions by default:

`fsbp-fix ec2.2 -profile <PROFILE>  [OPTIONAL_FLAGS]`